prow/config.yaml

```diff
@@ -43,6 +43,19 @@ deck:
   hidden_repos:
   - kubernetes-security
   google_analytics: UA-82843984-5
+  rerun_auth_config:
+    allow_anyone: false
+    authorized_users:
```

Comment:
Why can't we reuse the IsTrusted concept that we've already developed? AKA people who can trigger testing can rerun jobs

This seems like the most appropriate default

Reply:
That's one of the checks I plan on doing. But we can't use IsTrusted for batch jobs/periodics, so we'll have to have a whitelist regardless. See design doc for more details: 
https://docs.google.com/document/d/1ykgWeEnh4SwX9AuQz28sicFCpqjuEaYclLVTlStNggg/view#heading=h.43kyh5dm5cg6